libraries/src/MVC/View/AbstractView.php

```diff
@@ -239,7 +240,7 @@ public function getName()
             }
 
             if (empty($this->_name)) {
-                throw new \Exception(sprintf($this->text('JLIB_APPLICATION_ERROR_GET_NAME'), __METHOD__), 500);
+                throw new \Exception(Text::sprintf('JLIB_APPLICATION_ERROR_GET_NAME', __METHOD__), 500);
```

Comment:
```suggestion
                throw new \Exception(sprintf($this->getLanguage()->_('JLIB_APPLICATION_ERROR_GET_NAME'), __METHOD__), 500);
```

The abstractView has the LanguageAwareTrait which provides the `getLanguage()` available